src/functional/ffront/past_to_itir.py

```diff
@@ -123,7 +123,11 @@ def visit_Program(
     def _visit_stencil_call(self, node: past.Call, **kwargs) -> itir.StencilClosure:
         assert type_info.is_field_type_or_tuple_of_field_type(node.kwargs["out"].type)
 
-        output, domain = self._visit_stencil_call_out_arg(node.kwargs["out"], **kwargs)
+        if "field_domain" not in node.kwargs:
+            node.kwargs["field_domain"] = {}
+        output, domain = self._visit_stencil_call_out_arg(
+            node.kwargs["out"], node.kwargs["field_domain"], **kwargs
+        )
```

Comment:
Is this only for passing field_domain or an empty dict? Then it's much cleaner to do something like 
```suggestion
        field_domain = node.kwargs.get("field_domain", {})
        output, domain = self._visit_stencil_call_out_arg(
            node.kwargs["out"], field_domain, **kwargs
        )
```
instead of manipulating the node object.

Reply:
Actually, let's not pass an empty dict, but `None` in case the domain is not given.
Which reminds me that we should add a test with an empty domain, i.e. `field_op(..., out=..., domain={})`, and think about what the expected result is.